### DISCUSSION TITLE
Time typing bar plot changes every day

### DISCUSSION THREAD

**Discussion by Koushikphy** [monkeytypegame/monkeytype#3673] [Questions] [answered]
The bar plot in the account page that shows all the time/words for everyday. Yester day the data for 4th september showed  21 tests
![chrome_JLx580Cqts](https://user-images.githubusercontent.com/43264301/197102002-b93c12fa-c6f0-4b0c-99f5-933d5c8fe5c3.png)

But today the 4th september shows 19 tests
![chrome_KDzfbWGGfQ](https://user-images.githubusercontent.com/43264301/197102155-a0d195b9-2643-4146-93f1-4230ca772abb.png)

Consistently I am seeing the last bar shows wrong data every day.

**Comment by Miodec** [comment, marked as answer]
This will happen because of the 1000 result history limit. As you complete more tests, more tests go "out of the 1000" range, meaning the apparent time typing on that day will get shorter and shorter.
